include/user_functions/FlatDensityAuxFunction.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
```

Comment:
I'm pretty sure this was created when we were toying with using a perturbational pressure approach that needed a reference density field. None of the cases (that I'm adding as reg tests to exawind-driver) use this user_function.